package.json

```diff
@@ -26,14 +26,14 @@
   },
   "devDependencies": {
     "@ascendancyy/vue-cli-plugin-stylelint": "1.1.2",
-    "@storybook/addon-a11y": "5.1.11",
-    "@storybook/addon-actions": "5.1.11",
-    "@storybook/addon-backgrounds": "5.1.11",
-    "@storybook/addon-knobs": "5.1.11",
-    "@storybook/addon-storysource": "5.1.11",
-    "@storybook/addon-viewport": "5.1.11",
-    "@storybook/theming": "5.1.11",
-    "@storybook/vue": "5.1.11",
+    "@storybook/addon-a11y": "5.2.6",
```

Comment:
shall we add https://github.com/marketplace/dependabot-preview integration?

Reply:
NP with that, I think it's integrated by github anyway.